doc/source/development/contributing_environment.rst

```diff
@@ -36,29 +36,41 @@ These packages will automatically be installed by using the ``pandas``
 
 **Windows**
 
-You will need `Build Tools for Visual Studio 2019
+You will need `Build Tools for Visual Studio 2022
 <https://visualstudio.microsoft.com/downloads/>`_.
```

Comment:
can we link to

https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022

directly?

Reply:
Yes, we can link directly to BuildTools for VisualStudio2022.

Reply:
If we link directly, do we still need the below part of the instruction for installing a C compiler for Windows?

Reply:
as in, the warning "You DO NOT need to install Visual Studio 2022"?

might still be best to keep it, in case someone scrolls away from the exact location the link above

we could change it to `.. note::` though, "warning" doesn't really reflect the severity of the paragraph 😄